features/admins/admin_languages.feature

```diff
@@ -3,16 +3,49 @@ Feature: Manipulate languages on the Archive
   As as an admin
   I'd like to be able to manipulate languages on the Archive
 
+Scenario: An admin can add a language
+
+  Given basic languages
+    And I am logged in as an admin
+  When I go to the languages page
+    And I follow "Add a Language"
+    And I fill in "Name" with "Klingon"
+    And I fill in "Abbreviation" with "tlh"
+    And I press "Create Language"
+  Then I should see "Language was successfully added."
+    And I should see "The Archive supports these languages"
+    And I should see "Klingon"
+
 Scenario: Adding Abuse support for a language
+
   Given the following language exists
```

Comment:
Just... grammatically speaking -- I'm creating two languages, but the step says "language," singular.